src/ORM/FieldType/DBIp.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace SilverStripe\ORM\FieldType;
+
+use SilverStripe\ORM\FieldType\DBVarchar;
+use SilverStripe\Core\Validation\FieldValidation\IpFieldValidator;
+
+class DBIp extends DBVarchar
+{
+    private static array $field_validators = [
+        IpFieldValidator::class,
+    ];
```

Comment:
It's an edge case, but what if someone explicitly only wants IPv4 or IPv6?

Reply:
I've backlogged a new card https://github.com/silverstripe/silverstripe-framework/issues/11445